visualization/planning_scene_rviz_plugin/include/moveit/planning_scene_rviz_plugin/planning_scene_display.h

```diff
@@ -88,7 +88,7 @@ class PlanningSceneDisplay : public rviz::Display
   void queueRenderSceneGeometry();
 
   // pass the execution of this function call to a separate thread that runs in the background
-  void addBackgroundJob(const boost::function<void()> &job, const std::string &name);
+  void addBackgroundJob(const boost::function<void()> &job, const std::string &name, bool ownThread=false);
```

Comment:
+1 on a one-line `spawnBackgroundJob` function and a comment above that explains why this sometimes makes sense to use instead of `addBackgroundJob`.
I would be interested in a real explanation myself :-)